### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by AshutoshM29**
Added Product Page for the e-com website Urban Tails
Preview link: https://preview-urban-tails.netlify.app/pages/dog-products/treat

### CODE REVIEW

File: Pages/product.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+.section-filter {
+    max-width: 25vw;
+    border-right: 0.2rem solid var(--pink-nav);
+    position: sticky;
+    top: 6.5vh;
+    left: 0;
+    overflow: auto;
+    height: 100%;
+    padding: 1rem;
+    overflow: hidden;
```
Comment (sri-rishi) [inline Pages/product.css:28]: you are repeating overflow property

File: Pages/product.css [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    justify-content: space-evenly;
+}
+
+.card {
+    margin: 3rem 1rem;
+    border-radius: 0.25rem;
+    box-shadow: 0.1rem 0.1rem 0.5rem var(--gray-mild)
```
Comment (sri-rishi) [inline Pages/product.css]: use px for units less than 1rem

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by pradipgharge — COMMENTED** [review]
UI looks nice, good color theme. Code is also fine, no changes are required. Good to merge

File: Pages/product.css (hunk 2 above)
Comment (AshutoshM29) [reply Pages/product.css]: Is it mandatory??
and why should we use px??

**Review by manish-GH — APPROVED** [review]
Everything looks good, can be merged

File: Pages/product.css [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+.btn-wishlist {
+    display: flex;
+    border: none;
+    background-color: transparent;
+    font-size: 2rem;
+    position: absolute;
+    margin: 1.5rem 26.5rem;
+    color: var(--gray-mild);
+}
```
Comment (amantulsyan35) [inline Pages/product.css:186]: wishlist can be given a box shadow so that it is easy for the end user

**Comment by Sahil25061999** [comment]
Just a suggestion add to cart button can be given a different style such as outline or something, also reduce the opacity for card shadow.

File: Pages/product.css [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+    height: 100%;
+    padding: 1rem;
+    overflow: hidden;
+}
+
+.section-filter div {
+    display: flex;
+    align-items: center;
+    width: 25vw;
+}
```
Comment (Sahil25061999) [inline Pages/product.css]: you can propbably add width 100% since .section-filter is already 25vw

**Review by Sahil25061999 — APPROVED** [review]

File: Pages/product.css (hunk 2 above)
Comment (sri-rishi) [reply Pages/product.css]: Not mandatory but when your viewport changes rem value in px changed then you don't want these small size value to change. It's ok if you want them to change on changing viewport.

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
looks good , the side bar should have some margin and padding  as rating text are overlapping ,

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**